AutocompleteClient/FW/Logic/Query/Builder/AutocompleteQueryRequestBuilder.swift

```diff
@@ -10,10 +10,22 @@ import Foundation
 
 class AutocompleteQueryRequestBuilder: QueryRequestBuilder {
 
-    init(query: CIOAutocompleteQuery, autocompleteKey: String) {
+    init(query: CIOAutocompleteQuery, autocompleteKey: String, session: Int, userID: String?) {
         super.init(query: query.query, autocompleteKey: autocompleteKey)
         set(numResults: query.numResults)
         set(numResultsForSection: query.numResultsForSection)
+        set(session: session)
+        if let uID = userID{
+            self.set(userID: uID)
+        }
+    }
+    
+    func set(userID: String){
+        queryItems.append(URLQueryItem(name: "u", value: userID))
```

Comment:
The terminology we use in the JS autocomplete is `i` and `clientID`, as opposed to `u` and `userID` ... we should probably be consistent in how we use those terms.